target/linux/x86/base-files/lib/upgrade/platform.sh

```diff
@@ -94,4 +94,17 @@ platform_do_upgrade() {
 	#copy partition uuid
 	echo "Writing new UUID to /dev/$diskdev..."
 	get_image "$@" | dd of="/dev/$diskdev" bs=1 skip=440 count=4 seek=440 conv=fsync
+
+	local magic parttype=ext4
+	magic=$(dd if="/dev/$diskdev" bs=8 count=1 skip=64 2>/dev/null)
```

Comment:
Quotes might be a good idea here...

Reply:
I think he means around `magic="$(...)"`